src/strategies/authors/JekyllAuthorStrategy.ts

```
@@ -0,0 +1,15 @@
+import { AuthorStrategy } from './AuthorStrategy';
+import { FrontMatter } from '../../FrontMatterConverter';
+
+export class JekyllAuthorStrategy implements AuthorStrategy {
+  applyAuthors(frontMatter: FrontMatter, authors: string): FrontMatter {
+    const authorsList = authors.split(',').map(author => author.trim());
+    delete frontMatter.authors;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)